wave_kinematics/CMakeLists.txt

```diff
@@ -23,9 +23,4 @@ WAVE_ADD_TEST(
     tests/quadrotor_test.cpp
     tests/two_wheel_test.cpp
 )
-TARGET_LINK_LIBRARIES(
-    ${PROJECT_NAME}_tests
-    ${PROJECT_NAME}
-    wave_controls
-    wave_utils
-)
+TARGET_LINK_LIBRARIES(${PROJECT_NAME}_tests ${PROJECT_NAME})
```

Comment:
much cleaner with clients being able to access the previously linked libraries too